geniza/corpus/models.py

```diff
@@ -1103,6 +1104,10 @@ def index_data(self):
             "post_save": DocumentSignalHandlers.related_save,
             "pre_delete": DocumentSignalHandlers.related_delete,
         },
+        "annotations.annotation": {
+            "post_save": DocumentSignalHandlers.related_save,
+            "pre_delete": DocumentSignalHandlers.related_delete,
+        },
```

Comment:
awesome that this is all it takes, glad the existing signal handler is working so well